cypress/integration/eventplanner/editall.spec.js

```diff
@@ -0,0 +1,66 @@
+/// <reference types="cypress" />
+
+context('Editing', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000')
+  })
+  it('edits a packlist', () => {
+    cy.get('ul')
+      .contains('Rockharz')
+      .click({ timeout: 400 })
+      .get('img[alt="packlist"]')
+      .visit('http://localhost:3000/packlist/medieval')
+      .contains('PackList')
+      .get('button')
```

Comment:
I think you don't necesseraly need the element type ('ul', 'button') if the text is unique. You only need to test the element type if you have some special requirements. The user doesn't care if it's a ul or some other element. Then your test would be shorter.

Reply:
My buttons are img so I dont have any texts on my elements..